modules/ROOT/pages/gov-cloud-standalone.adoc

```diff
@@ -0,0 +1,42 @@
+= Standalone Mule Support in MuleSoft Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MuleSoft Government Cloud deployment permits deployment of Mule runtime engines into an on-premises environment (standalone Mules). Standalone Mules, and
+MuleSoft support for standalone Mules, are outside the scope of FedRAMP. The customer is responsible for ensuring that all data shared with Salesforce complies
+with any and all data and information protection policies, requirements, and standards. Review the MuleSoft System Security Plan for FedRAMP scope.
+
+Rigorous internal security and compliance reviews were performed on services that support standalone Mule deployments in MuleSoft Government Cloud. 
+The Significant Change Request Form and updated System Security Plan were approved by our FedRAMP sponsoring agency.
+
+For FIPS compliance, Mule 4 agent version 2.4.9 or later is required.
```

Comment:
"For FIPS compliance, Mule Runtime version 4.1.x or later, along with runtime agent version 2.4.9 or later is required."

Reply:
Updated this.